interfaces/builtin/basedeclaration.go

```diff
@@ -203,6 +203,14 @@ slots:
       slot-snap-type:
         - core
     deny-auto-connection: true
+  dbus:
+    allow-installation:
+      slot-snap-type:
+        - app
+    deny-connection:
+      slot-attributes:
+        name: .+
+    deny-auto-connection: true
```

Comment:
Yes. I thought we agreed that people would need to have their well-known dbus name approved in the store. This will trigger a manual review and enforce the use of a snap declaration to claim the well-known name.